src/libNetwork/P2PComm.cpp

```diff
@@ -691,6 +737,43 @@ void P2PComm::EventCallback(struct bufferevent* bev, short events,
   }
 }
 
+void P2PComm::EventCbServerSeed(struct bufferevent* bev, short events,
+                                [[gnu::unused]] void* ctx) {
+  LOG_MARKER();
```

Comment:
Maybe you could call this only if log level is DEBUG

Reply:
Removed `LOG_MARKER`